src/packets/internal.rs

```diff
@@ -87,22 +86,22 @@ pub enum DHTRequestSpecific {
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
 #[serde(untagged)] // This means order matters! Order these from most to least detailed
 pub enum DHTResponseSpecific {
-    DHTGetPeersResponse {
+    GetPeers {
         #[serde(rename = "r")]
         arguments: DHTGetPeersResponseArguments,
     },
 
-    DHTSampleInfoHashesResponse {
+    SampleInfoHashes {
         #[serde(rename = "r")]
         arguments: DHTSampleInfoHashesResponseArguments,
     },
 
-    DHTFindNodeResponse {
+    FindNode {
         #[serde(rename = "r")]
         arguments: DHTFindNodeResponseArguments,
     },
 
-    DHTPingResponse {
+    PingResponse {
```

Comment:
Please remove `Response` from the end of this variant to be consistent with the others.

```suggestion
    Ping {
```

Reply:
Done